### PR TITLE
New join mockup

### PR DESCRIPTION
**Pull request by june-andronick**

### CODE REVIEW

**Comment by gernotheiser** [comment]
About and Join pages look fine, I'm not sure I like the mixture of fonts and colours on the first line of the membership page

**Comment by june-andronick** [comment]
This is just to have the "join now" more visible (I'll work on the rest this afternoon).
Drop down menus have also been adjusted.
I'm not sure what to fix for the remaining broken link yet...

**Comment by lsf37** [comment]
I think the remaining Link checker failure can be ignored. It looks like it's reacting to an internal warning only. I might be able to fix that later today.

**Comment by lsf37** [comment]
We used a different highlight colour on the front page -- the darker yellow, because people complained that the bright one is too hard to read on white (which I agree with, it works well on black, but not well on white).  

Shall we do the same for here? It's CSS class `highlight`

File: content/css/sel4.css [hunk 1]
```diff
@@ -266,4 +266,15 @@ a.divLink {
   color:#B78B41;  /* emphasisDark */
   font-size:15px;
   font-weight:bold;
-}
\ No newline at end of file
+}
+
+.emph {
+    color:#FFCA5F;
```
Comment (lsf37) [inline content/css/sel4.css:272]: I'd try this one here:

```suggestion
    color:#B78B41;  /* emphasisDark */
```

File: content/css/sel4.css [hunk 2]
```diff
@@ -266,4 +266,15 @@ a.divLink {
...
+}
+
+
+.join {
+    color:#FFCA5F;
```
Comment (lsf37) [inline content/css/sel4.css:277]: (same as above)

```suggestion
    color:#B78B41;  /* emphasisDark */
```

**Comment by gernotheiser** [comment]
I agree with the colour being too light, but I don't seem to see a change, the preview still seems to show the lighter colour?

**Comment by gernotheiser** [comment]
Also, I think the font mixture in the "add your name" para looks less weird if you simply force it on a new line.

**Comment by june-andronick** [comment]
Like Gernot, I'm not sure how to visualise your suggested change Gerwin.
(Also: I find this brown-ish quite ugly, but will live with it if needed)
Taking about colours: I think the blue for links is hardly visible...

**Comment by june-andronick** [comment]
Gernot: I've already changed the "add your name" part.

**Comment by gernotheiser** [comment]
> Gernot: I've already changed the "add your name" part.

Yes, I can see it in the diff, but the rendering doesn't reflect it, weird

**Comment by lsf37** [comment]
> I agree with the colour being too light, but I don't seem to see a change, the preview still seems to show the lighter colour?

Yes, I didn't actually change anything yet, just made a suggestion. @june-andronick can accept it with the "Commit suggestion" button.

**Comment by gernotheiser** [comment]
OK, I finally can see it (no idea why not earlier).
I'm fine modulo resolving the highlight colour.

**Comment by lsf37** [comment]
> Taking about colours: I think the blue for links is hardly visible...

Yes, see also #31 -- we need a new shade in the scheme. The problem is not actually simple: if you just make it lighter, it becomes unreadable against the white background, and if you leave it dark so there is good contrast against the background, it becomes hard to distinguish against the rest of the text. If you make it a colour, it's not good for accessibility, and it also draws too much attention to the links -- you don't want them to be too highlighted either, just recognisable as links.

In short, that one is a separate discussion.

**Comment by gernotheiser** [comment]
Agree re link colour being a separate issue.

**Comment by lsf37** [comment]
With new highlight colour now [here](https://htmlpreview.github.io/?https://raw.githubusercontent.com/seL4/website_pr_hosting/PR_39/localhost/Foundation.html)

File: content/css/sel4.css [hunk 3]
```diff
@@ -266,4 +266,15 @@ a.divLink {
   color:#B78B41;  /* emphasisDark */
   font-size:15px;
   font-weight:bold;
-}
\ No newline at end of file
+}
+
+.emph {
+    color:#B78B41;  /* emphasisDark */
```
Comment (lsf37) [inline content/css/sel4.css:272]: Trying out a different one:
```suggestion
    color:#668B37;  /* greenDark */
```

File: content/css/sel4.css [hunk 4]
```diff
@@ -266,4 +266,15 @@ a.divLink {
...
+}
+
+
+.join {
+    color:#B78B41;  /* emphasisDark */
```
Comment (lsf37) [inline content/css/sel4.css:277]:
```suggestion
    color:#668B37;  /* greenDark */
```

**Comment by gernotheiser** [comment]
LGTM. @june-andronick?

**Review by lsf37 — APPROVED** [review]
The green looks much better.

**Comment by gernotheiser** [comment]
looking at the CI failures:
- not sure what's wrong with the signoff – June's email address invalid? But looks like her last commit's weren't actually signed off

**Comment by gernotheiser** [comment]
- the linklint is a unconditional failure, so has nothing to do with the changes
- "unexpected token" in the HTML checker might be a unicode character somewhere

**Comment by lsf37** [comment]
> looking at the CI failures:
> 
> * not sure what's wrong with the signoff – June's email address invalid? But looks like her last commit's weren't actually signed off

Yes, that's from accepting the commit suggestion on the github interface -- if you don't manually add the sign-off, it remains unsigned. It's fine, though, we squashed all commits into one for the merge and used a sign-off for that.
